### PR TITLE
No more path rewriting

### PR DESCRIPTION
**Pull request by eyakubovich**
Path rewriting causes too many problems when vendoring
vendored code. When CNI code is vendored into rkt,
godep has problems code already vendored by CNI.

### CODE REVIEW

File: build [hunk 1]
```diff
@@ -9,7 +9,7 @@ if [ ! -h gopath/src/${REPO_PATH} ]; then
 fi
 
 export GOBIN=${PWD}/bin
-export GOPATH=${PWD}/gopath
+export GOPATH=${PWD}/gopath:`pwd`/Godeps/_workspace
```
Comment (jonboulle) [inline build]: Godeps should be first, surely

Comment (jonboulle) [inline build]: also backtick? what is this 1999?

Comment (eyakubovich) [inline build]: Why Godeps first? Wouldn't you want code local to this repo to be searched before dependencies?

Comment (jonboulle) [inline build]: I misread slightly, thought it was including predefined GOPATH. In this
case, either option seems equally unintuitive to me, so this is fine :P

**Comment by jonboulle** [comment]
ok

**Pull request merged**